server/boot/user.js

```diff
@@ -183,6 +185,12 @@ module.exports = function(app) {
         }
         profileUser = profileUser.toJSON();
 
+        // timezone of signed-in account
+        // to show all date related components
+        // using signed-in account's timezone
+        // not of the profile she is viewing
+        const timezone = req.user.timezone || 'UTC';
```

Comment:
`Cannot read property timezone of undefined`

You'll need to verify that the user exists as you don't need to be signed to hit this endpoint